src/c++/perf_analyzer/test_profile_data_exporter.cc

```diff
@@ -244,4 +248,114 @@ TEST_CASE("profile_data_exporter: OutputToFile")
   }
 }
 
+TEST_CASE("profile_data_exporter: AddServiceKind")
+{
+  MockProfileDataExporter exporter{};
+  exporter.ClearDocument();
```

Comment:
Why call this? Shouldn't it be empty when initialized above?

Reply:
`document_` is null initially so needs to be set to Object.

Reply:
Ah, then the ClearDocument function name seems a little misleading. I thought it was clearing the document (i.e. "clearing"/removing stuff). It sounds like it's initializing it to an object.